server/src/main/java/org/apache/iotdb/db/qp/physical/sys/CreateMultiTimeSeriesPlan.java

```diff
@@ -144,6 +144,11 @@ public void setIndexes(List<Integer> indexes) {
     return results;
   }
 
+  @Override
+  public List<PartialPath> getPrefixPaths() {
+    return null;
```

Comment:
Never return `null`! You can use `Collections.emptyList()`.BTW, why not return true `prefixPaths` for `CreateMultiTimeSeriesPlan` ?

Reply:
The method get prefixPaths is defined for batch plan. But CreateMultiTimeSeriesPlan stores each path of time series, I think it is no need to implement it.

Reply:
So please return `Collections.emptyList()` rather than `null`.There is no doubt that returning`null` is not elegant.